modules/cnf-topology-aware-lifecycle-manager-PAO-update.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// Epic CNF-4767 (4.11), Story TELCODOCS-593
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="talo-PAO-update_{context}"]
+= Removing Performance Addon Operator subscriptions from deployed clusters
+
+In earlier versions of {product-title}, the Performance Addon Operator was used to implement automatic tuning to achieve low latency performance for OpenShift applications. In {product-title} 4.11, these functions are part of the Node Tuning Operator. 
+
+Do not install the Performance Addon Operator on clusters running {product-title} 4.11 or later. When you upgrade to {product-title} 4.11,  the Node Tuning Operator automatically removes the Performance Addon Operator. However, you  need to manually remove any Performance Addon Operator subscriptions to prevent a reinstallation of the operator. The reference DU profile includes the Performance Addon Operator in the `common-ranGen.yaml` `PolicyGenTemplate`. To remove the subscription from deployed spoke clusters you must update this policy.
+
+.Prerequisites
+
+* Create a Git repository where you manage your custom site configuration data.
+* Update to {product-title} 4.11. 
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Change the `complianceType` to `mustnothave` for the Performance Addon Operator namespace, operator group and subscription in the `common-ranGen.yaml` file.
++
+[source,yaml]
+----
+ -  fileName: PaoSubscriptionNS.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+ -  fileName: PaoSubscriptionOperGroup.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+ -  fileName: PaoSubscription.yaml
+    policyName: "subscriptions-policy"
+    complianceType: mustnothave
+----
++
+. Merge the changes with your custom site repository and wait for the ArgoCD application to synchronize the change to the hub cluster. The state of the common policy changes to `Non-Compliant`.
+. Apply the change to your target clusters using the Topology Aware Lifecycle Operator. For more information about rolling out configuration changes, see the _Additional resources_ section.
```

Comment:
Use quotes instead of underscores to refer to other sections that we can't link to.

s/_Additional resources_/"Additional resources"